test/boulder-config.json

```diff
@@ -76,7 +76,12 @@
               {
                 "ID": "1.3.6.1.4.1.44947.1.1.1",
                 "type": "id-qt-cps",
-                "qualifier": "http://cps.root-x1.letsencrypt.org"
+                "qualifier": "http://cps.letsencrypt.org"
+              },
+              {
+                "ID": "1.3.6.1.4.1.44947.1.1.1",
+                "type": "id-qt-unotice",
+                "qualifier": "This Certificate may only be relied upon by Relying Parties and only in accordance with the Certificate Policy found at https://letsencrypt.org/documents/"
```

Comment:
As produced in the dump from @jcjones, the bad thing is happening:
```
            Policy: 1.3.6.1.4.1.44947.1.1.1
              CPS: http://cps.letsencrypt.org
            Policy: 1.3.6.1.4.1.44947.1.1.1
              User Notice:
                Explicit Text: This Certificate may only be relied upon by Relying Parties and only in accordance with the Certificate Policy found at https://letsencrypt.org/documents/
```
This needs to be fixed.